Content.Server/Abilities/Mime/MimePowersSystem.cs

```diff
@@ -80,22 +78,13 @@ private void OnInvisibleWall(EntityUid uid, MimePowersComponent component, Invis
             if (tile == null)
                 return;
 
-            // Check there are no walls there
-            if (_turf.IsTileBlocked(tile.Value, CollisionGroup.Impassable))
+            // Check if the tile is blocked by a wall or mob, and don't create the wall if so
+            if (_turf.IsTileBlocked(tile.Value, CollisionGroup.MobLayer | CollisionGroup.WallLayer))
```

Comment:
```suggestion
            if (_turf.IsTileBlocked(tile.Value, CollisionGroup.Impassable | CollisionGroup.Opaque))
```
Ok, I did some more testing and I think it should be this instead. The Impassable layer is walls and such (we had that part before) and the Opaque layer is making sure we cannot put it over mobs (Opaque is part of MobLayer), but we can still place it over tables and machines like before.

Reply:
Done.